ui/perfherder/compare/ComparePageTitle.jsx

```diff
@@ -13,6 +14,7 @@ export default class ComparePageTitle extends React.Component {
       inEditMode: false,
       pageTitle: props.pageTitleQueryParam || props.title,
       newPageTitle: props.pageTitleQueryParam || props.title,
+      tabTitle: props.defaultPageTitle,
```

Comment:
The initial value from here will be irelevant *(it'll be an empty string)*.
We're better off with setting this to `null`.

Reply:
Thanks for your help @ionutgoldan 
I have made the changes